### PR TITLE
fix(TagPicker): focus outline for TagItem remove button now meets required contrast ratio

### LINKED ISSUE #24100: [Bug]: A11y bug in TagPicker
### Library

Web Components (@fluentui/web-components)

### System Info

```shell
System:
    OS: Windows 10 10.0.19044
    CPU: (4) x64 Intel(R) Core(TM) i7-7600U CPU @ 2.80GHz
    Memory: 13.30 GB / 31.89 GB
  Browsers:
    Chrome: 103.0.5060.134
    Edge: Spartan (44.19041.1266.0), Chromium (103.0.1264.71)
    Internet Explorer: 11.0.19041.1566
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://developer.microsoft.com/en-us/fluentui#/controls/web/pickers

## Actual Behavior
Hi FluentUI team, we have an accessibility issue raised in the tagPicker where the focus on the 'remove button' is not having enough contrast to be visible (see image attached) when navigating via tabs. Please let me know in case anyone know how to resolve this issue.
Ideally, if there is any way to override the color of tagitem on focus would be useful.

b105f76a-991d-4523-9aee-024966b1b5ff

## Expected Behavior
A way to override the style of focus border of TagItem

### Logs

_No response_

### Requested priority

High

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by TristanWatanabe**
## Current Behavior

- Remove button on a `TagItem` has a focus outline with a contrast ratio of 1.425: 1 and does not meet minimum contrast ratio of 3:1.

TagItemFocusIssue

## New Behavior
- Remove button on a `TagItem` focus outline now meets minimum contrast ratio of 3:1. Changed the outline color to white, similar to `PrimaryButton`'s keyboard focus outline color. 

TagItemFocusFix

## Related Issue(s)

Fixes #24100

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/29769d9e95a0395c36d596e4e186744da2d232d4'>29769d9e95a0395c36d596e4e186744da2d232d4</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/BaseButton-mount_base.html">803</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/BaseButton-mount.html">815</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Breadcrumb-mount_base.html">2385</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Breadcrumb-mount.html">2384</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Checkbox-mount_base.html">2267</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Checkbox-mount.html">2245</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/CheckboxBase-mount_base.html">1943</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/CheckboxBase-mount.html">1962</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/ChoiceGroup-mount_base.html">4092</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/ChoiceGroup-mount.html">4072</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/ComboBox-mount_base.html">854</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/ComboBox-mount.html">841</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/CommandBar-mount_base.html">9267</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/CommandBar-mount.html">9190</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/ContextualMenu-mount_base.html">10559</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/ContextualMenu-mount.html">10614</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/DefaultButton-mount_base.html">979</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/DefaultButton-mount.html">1020</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/DetailsRow-mount_base.html">3295</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/DetailsRow-mount.html">3350</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/DetailsRowFast-mount_base.html">3354</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/DetailsRowFast-mount.html">3294</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/DetailsRowNoStyles-mount_base.html">3158</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/DetailsRowNoStyles-mount.html">3135</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Dialog-mount_base.html">2454</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Dialog-mount.html">2474</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/DocumentCardTitle-mount_base.html">155</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/DocumentCardTitle-mount.html">152</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Dropdown-mount_base.html">2903</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Dropdown-mount.html">2911</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/FocusTrapZone-mount_base.html">1647</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/FocusTrapZone-mount.html">1625</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/FocusZone-mount_base.html">1620</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/FocusZone-mount.html">1594</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/IconButton-mount_base.html">1515</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/IconButton-mount.html">1518</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Label-mount_base.html">302</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Label-mount.html">315</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Layer-mount_base.html">2748</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Layer-mount.html">2770</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Link-mount_base.html">419</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Link-mount.html">429</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/MenuButton-mount_base.html">1282</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/MenuButton-mount.html">1284</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/MessageBar-mount_base.html">1831</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/MessageBar-mount.html">1828</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Nav-mount_base.html">2835</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Nav-mount.html">2873</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/OverflowSet-mount_base.html">953</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/OverflowSet-mount.html">947</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Panel-mount_base.html">1903</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Panel-mount.html">1931</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Persona-mount_base.html">894</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Persona-mount.html">880</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Pivot-mount_base.html">1260</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Pivot-mount.html">1258</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/PrimaryButton-mount_base.html">1142</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/PrimaryButton-mount.html">1112</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Rating-mount_base.html">6639</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Rating-mount.html">6760</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/SearchBox-mount_base.html">1118</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/SearchBox-mount.html">1127</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Shimmer-mount_base.html">2146</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Shimmer-mount.html">2155</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Slider-mount_base.html">1692</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Slider-mount.html">1684</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/SpinButton-mount_base.html">4404</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/SpinButton-mount.html">4380</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Spinner-mount_base.html">394</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Spinner-mount.html">385</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/SplitButton-mount_base.html">2775</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/SplitButton-mount.html">2742</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Stack-mount_base.html">447</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Stack-mount.html">456</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/StackWithIntrinsicChildren-mount_base.html">1994</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/StackWithIntrinsicChildren-mount.html">2016</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/StackWithTextChildren-mount_base.html">4583</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/StackWithTextChildren-mount.html">4521</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/SwatchColorPicker-mount_base.html">10055</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/SwatchColorPicker-mount.html">10137</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/TagPicker-mount_base.html">2372</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/TagPicker-mount.html">2341</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/TeachingBubble-mount_base.html">85184</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/TeachingBubble-mount.html">86205</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Text-mount_base.html">383</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Text-mount.html">363</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/TextField-mount_base.html">1213</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/TextField-mount.html">1226</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/ThemeProvider-mount_base.html">1041</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/ThemeProvider-mount.html">1050</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/ThemeProvider-virtual-rerender_base.html">579</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/ThemeProvider-virtual-rerender.html">582</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1527</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1552</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Toggle-mount_base.html">697</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/Toggle-mount.html">692</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/buttonNative-mount_base.html">117</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24126/perf-test/buttonNative-mount.html">106</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Review by smhigley — APPROVED** [review]
Other than Micah’s question, it looks good to me. It’s technically not an a11y issue since the blue background of the entire tag functions as the focus style, but since this has come up twice now we might as well update it :P

File: packages/react/src/components/pickers/TagPicker/TagItem.styles.ts [hunk 1]
```diff
@@ -93,6 +93,7 @@ export function getStyles(props: ITagItemStyleProps): ITagItemStyles {
     ],
     close: [
       classNames.close,
+      getFocusStyle(theme, { borderColor: 'transparent', inset: 1, outlineColor: palette.white }),
```
[the remark this answers is not in the document]
Comment (TristanWatanabe) [reply packages/react/src/components/pickers/TagPicker/TagItem.styles.ts:96]: No strong reason other than the `getFocusStyle` function has a default border color of white and I simply wanted to keep all focus styling the same as before except for the outline color being white now. Everything in high contrast mode and dark mode should look the same as before

**Review by micahgodbolt — APPROVED** [review]

**Pull request merged**